src/centralstore/incremental/NodeManager.cpp

```diff
@@ -133,6 +134,8 @@ NodeBlock *NodeManager::addNode(std::string nodeId) {
 }
 
 RelationBlock *NodeManager::addEdge(std::pair<std::string, std::string> edge) {
+    std::unique_lock<std::mutex> guard(lockEdgeAdd);
+    // lockEdgeAdd.lock();
```

Comment:
Lets remove this commented line

Reply:
Removed the commented text https://github.com/miyurud/jasminegraph/pull/132/commits/d8d887f5453408266df16065d675be6691184b72

Reply:
Ok